### PR TITLE
use_aws_shared_config_files

### PR DESCRIPTION
**Pull request by zeddosx**
https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_configuration.html#use-aws-shared-config-files

Option to fix this warning.

image

### CODE REVIEW

File: src/DI/AwsSdkNetteExtension.php [hunk 1]
```diff
@@ -19,7 +19,8 @@ class AwsSdkNetteExtension extends Nette\DI\CompilerExtension
 		'credentials' => [
 			'key' => NULL,
 			'secret' => NULL
-		]
+		],
+        'use_aws_shared_config_files' => true
```
Comment (paveljanda) [inline src/DI/AwsSdkNetteExtension.php]: Could you please change your indentation to tabs?

Comment (zeddosx) [reply src/DI/AwsSdkNetteExtension.php]: I changed it, is it okay now?

**Comment by paveljanda** [comment]
Cool, thanks

**Comment by paveljanda** [comment]
https://github.com/ublaboo/aws-sdk-nette-extension/releases/tag/v1.1.0

**Pull request merged**